DuckDuckGo/Subscription/UserScripts/SubscriptionPagesUseSubscriptionFeature.swift

```diff
@@ -340,65 +351,71 @@ final class SubscriptionPagesUseSubscriptionFeature: Subfeature, ObservableObjec
     }
 
     func backToSettings(params: Any, original: WKScriptMessage) async -> Encodable? {
-        guard let accessToken = accountManager.accessToken else {
-            Logger.subscription.error("Missing access token")
-            return nil
-        }
-
-        switch await accountManager.fetchAccountDetails(with: accessToken) {
-        case .success(let accountDetails):
-            switch await subscriptionManager.subscriptionEndpointService.getSubscription(accessToken: accessToken) {
-            case .success:
-                accountManager.storeAccount(token: accessToken,
-                                            email: accountDetails.email,
-                                            externalID: accountDetails.externalID)
-                onBackToSettings?()
-            case .failure(let error):
-                Logger.subscription.error("Error retrieving subscription details: \(error.localizedDescription)")
-            }
-        case .failure(let error):
-            Logger.subscription.error("Could not get account Details: \(error.localizedDescription)")
-            setTransactionError(.generalError)
-        }
+        Logger.subscription.log("Back to settings")
+//        guard let accessToken = accountManager.accessToken else {
+//            Logger.subscription.error("Missing access token")
+//            return nil
+//        }
+//
+//        switch await accountManager.fetchAccountDetails(with: accessToken) {
+//        case .success(let accountDetails):
+//            switch await subscriptionManager.subscriptionEndpointService.getSubscription(accessToken: accessToken) {
+//            case .success:
+//                accountManager.storeAccount(token: accessToken,
+//                                            email: accountDetails.email,
+//                                            externalID: accountDetails.externalID)
+//                onBackToSettings?()
+//            case .failure(let error):
+//                Logger.subscription.error("Error retrieving subscription details: \(error.localizedDescription)")
+//            }
+//        case .failure(let error):
+//            Logger.subscription.error("Could not get account Details: \(error.localizedDescription)")
+//            setTransactionError(.generalError)
+//        }
+//        return nil
```

Comment:
1. Probably commented out code can be removed.
2. What about handling the error scenarios like in original code e.g. L:373?